python/ray/air/BUILD

```diff
@@ -232,4 +232,10 @@ py_test(
 py_library(
     name = "ml_lib",
     srcs = glob(["**/*.py"], exclude=["tests/*.py"]),
+    visibility = [
+        "//python/ray/air:__pkg__",
+        "//python/ray/air:__subpackages__",
+        "//python/ray/train:__pkg__",
+        "//python/ray/train:__subpackages__",
```

Comment:
For my own understanding, what does this do?

Reply:
i noticed that ray train tests are actually using modules from air folder, such as the `test_horovod`. So I setup both ML and AIR bazel targets to be visible to each other so tests and directly import and use if needed.